src/status_im/ui/screens/home/views/inner_item.cljs

```diff
@@ -16,42 +16,30 @@
             [taoensso.timbre :as log]
             [status-im.ui.components.chat-icon.screen :as chat-icon.screen]))
 
-(defn message-content-text [{:keys [content] :as message}]
-  (reagent/create-class
-    {:display-name "message-content-text"
-     :component-will-mount
-     #(when (and (or (:command content)
-                     (:content-command content))
-                 (not (:short-preview content)))
-        (re-frame/dispatch [:request-command-message-data message
-                            {:data-type   :short-preview
-                             :cache-data? true}]))
-     :reagent-render
-     (fn [{:keys [content] :as message}]
-       [react/view styles/last-message-container
-        (cond
+(defn message-content-text [{:keys [content] :as message}] 
+  [react/view styles/last-message-container
+   (cond
 
-          (not message)
-          [react/text {:style styles/last-message-text}
-           (i18n/label :t/no-messages)]
+     (not message)
+     [react/text {:style styles/last-message-text}
```

Comment:
Out of PR, but this duplication of similar `react/text` block could be improved.